starter-code/css/main.css

```diff
@@ -1,8 +1,139 @@
-/*
-Colors:
+body {
+    width: 100%;
+    height: 100%;
+    margin: 0px;
+    padding: 0px;
+    overflow-x: hidden;
+    font-family: Circular, Helvetica, Arial, sans-serif;
+ }
 
-Text: 1A1A1A
-Green: #00B172
-White: #FFF
+header {
+    height: 40px;
+}
 
-*/
+header img {
+    width: 230px;
+    margin-left: 20px;
+    margin-top: 14px;
+}
+
+header nav {
+    float: right;
+    margin-right: 20px;
+    margin-top: 16px;
+}
+
+nav ul li {
+    display: inline-block;
+    padding: 0 10px;
+    font-size: 20px;
+}
+
+nav ul li a {
+    text-decoration: none;
+    color: #1A1A1A;
+}
+
+#hero {
+    background-image: url("../images/landing.jpg");
+    background-size: cover;
+    background-position: bottom; 
+    height: 800px;
+    color: #FFF;
+    text-align: center;
+}
+
+#hero h1 {
+    font-size: 75px;
+    font-weight: 600;
+    letter-spacing: 1px;
+    padding-top: 200px;
+}
+
+#hero h3 {
+    font-size: 30px;
+    font-weight: 300;
+    margin-top: -30px
+}
+
+h2 {
+    border-bottom: #00B172 4px solid;
+    width: 320px;
+    margin: 0 auto;
+    font-size: 35px;
+    font-weight: 600;
+    padding-bottom: 5px;
+    margin-top: 50px;
+}
+
+#whats-on {
+    text-align: center;
+}
+
+#whats-on img {
+    width: 135px;
+    height: 100px;
+}
+
+#whats-on div {
+    width: 33%;
+    float: left;
+    height: 400px;
+    margin-top: 100px;
+}
+
+#whats-on h3 {
+    font-size: 27px;
+    color: #00B172;
+}
+
+#whats-on p {
+    font-size: 23px;
+    font-weight: 100;
+}
+
+#green-stuff {
+    background-color: #00B172;
+    height: 800px;
+    width: 90%;
+    position: absolute;
+    top: 1500px;
+    margin: 30px;
+    background-image: url("../images/spotify-icon-white.png");
+    background-repeat: no-repeat;
+    background-size: 130px;
+    background-position: center; 
+    color: #FFF;
+    padding-left: 70px;
+}
+
+#green-stuff img {
+    width: 300px;
+    position: absolute;
+    top: 130px;
+    left: 800px;
+}
```

Comment:
Not sure `position: absolute` is ideal here, since the image will stay in the same place in relation to your `#green-stuff` section.  Since you have positioned the `img` from the left it has funny consequences when the width of your screen is smaller.  Perhaps try using the `vw` (view width) unit for a most responsive layout.  And I would say if the image is closer to the right, set position from the right so that if anything gets cut off its the empty bit in the middle